### DISCUSSION TITLE
Install Package Nginx Signature GPG error for version 4.0.3

### DISCUSSION THREAD

**Discussion by chrisho95** [netbox-community/netbox-docker#1266] [Getting Help] [answered]
### Current Behavior

In my Dockerfile, I get the v4.0.3 image and do an atp-get update and get the following error message.

####################################
FROM netboxcommunity/netbox:v4.0.3

RUN apt-get update
####################################

Errors:

![image](https://github.com/netbox-community/netbox-docker/assets/82440344/914a7451-ed39-4c45-874f-83e8d56b55ed)

### Expected Behavior

In my Dockerfile, I get the v4.0.3 image and do an atp-get update and get the following error message.

####################################
FROM netboxcommunity/netbox:v4.0.3

RUN apt-get update
####################################

Errors:

![image](https://github.com/netbox-community/netbox-docker/assets/82440344/914a7451-ed39-4c45-874f-83e8d56b55ed)

### Docker Compose Version

Docker Compose version v2.23.0

### Docker Version

```text
Client: Docker Engine - Community
 Version:           25.0.3
 API version:       1.44
 Go version:        go1.21.6
 Git commit:        4debf41
 Built:             Tue Feb  6 21:14:26 2024
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          25.0.3
  API version:      1.44 (minimum version 1.24)
  Go version:       go1.21.6
  Git commit:       f417435
  Built:            Tue Feb  6 21:14:26 2024
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.28
  GitCommit:        ae07eda36dd25f8a1b98dfbf587313b99c0190bb
 runc:
  Version:          1.1.12
  GitCommit:        v1.1.12-0-g51d5e94
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### The git Revision

N/A

### The git Status

```text
N/A
```

### Startup Command

docker compose up -d

### NetBox Logs

```text
![image](https://github.com/netbox-community/netbox-docker/assets/82440344/914a7451-ed39-4c45-874f-83e8d56b55ed)
```

### Content of docker-compose.override.yml

```yaml
version: '3.4'
services:
  netbox:
    ports:
    - '8080:8080'
```

**Comment by tobiasge** [comment, marked as answer]
The key was changed by Nginx a few days ago. You can run 
```bash
curl --silent --output /usr/share/keyrings/nginx-keyring.gpg \
      https://unit.nginx.org/keys/nginx-keyring.gpg
```
to get the current key.

**Reply by chrisho95** [reply]
Thx Tobiasge, it's Ok Now.
